gym/spaces/dict.py

```diff
@@ -134,9 +136,9 @@ def __len__(self) -> int:
 
     def __repr__(self) -> str:
         return (
-            "Dict("
-            + ", ".join([str(k) + ":" + str(s) for k, s in self.spaces.items()])
-            + ")"
+            "Dict({"
+            + ", ".join([str(k) + ": " + str(s) for k, s in self.spaces.items()])
```

Comment:
I'd suggest removing such changes, since they don't really have anything to do with the issue you're solving

Reply:
Very true, I will remove it, I planning on creating a PR with typing for batched_space where it means more sense there